test/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-static" version="1.8.1" targetFramework="native" />
+  <package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-dyn" version="1.8.0" targetFramework="native" />
```

Comment:
For reference, the `rt-static`/`rt-dyn` NuGet package to use should (I believe) match the `RuntimeLibrary` setting in the project file. The `RuntimeLibrary` setting in the test project should match the `RuntimeLibrary` setting in the NAS2D project.

Basically, all compiled modules which are linked together (gtest, test project, NAS2D) should either all link to the static version of the runtime, or they should all link to the dynamic version of the runtime.

If you link to the dynamic version, then the library will have a dependency on the MSVCRT runtime DLL. If you link to the static version, the compiled output may be slightly larger, but will be standalone.

I have a preference for static linking. When it comes time for deployment, end users are never forced to install MSVCRT redistributable packages. In the larger picture, it also avoids the Windows DLL Hell problem. I've heard that Google uses static linking for everything, which I believe stems from that.

We may want to check with @ldicker83 on this one, as it has implications for downstream projects and distribution. Namely OPHD.

Reply:
Agreed on static linking being a little easier to integrate against outside of Windows environment. I vaguely remember we have run into trouble on missions not being compatible on Wine when not statically linked.

I would be careful about saying slightly larger builds for static. I noticed op2ext was reset from dynamic to static linking since the last release (no sure exactly why or when). Dynamically, it compilied around 65 KB with an associated PDB file of about 1,620KB. Statically, it now compiles at 577KB with an associated PDB file of about 9,748kb. That is closing on a factor of 10 increase. Fortunately both files zip reasonably well.

For NAS2D, I switched from static to dynamic for the test project to match the 2017 project being dynamic already in addition to being Microsoft's recommendation. Sorry I wasn't clear about that in my PR comments. Happy to switch to whatever the project file is updated to.

Reply:
Good to point out the inconsistency between the main project file and the test project file.

You're right, the inflation in code size can seem dramatic when you count it proportionally. I suppose to be fair, you'd need to compare the size increase with the size of the CRT itself. (Though it may actually still be out of proportion with the size of the CRT). Still, the benefits may outweigh the size increase, which in overall numbers, doesn't seem to be a huge concern for users these days. In particular, users of older versions of Windows, or Wine, shouldn't need to install a CRT redistributable package.

Not sure about the Microsoft recommendation part. Is that in an error/warning message, or posted online somewhere? I'm guessing from their perspective, they're probably assuming an up-to-date version of Windows.

----

I suppose we could offer multiple configurations. Not sure if we want to go down that path though.

Reply:
@DanRStevens,

See image below for where recommendation to use dynamic comes from. I'm happy with static or dynamic.

<img width="373" alt="google test setup dialog" src="https://user-images.githubusercontent.com/15183337/56249682-a90d2b80-607a-11e9-8100-9f3fb8d1bf22.png">